Rakefile

```diff
@@ -13,6 +14,18 @@ task :changelog, [:version] do |_t, args|
   sh "./scripts/generate_changelog.rb #{args[:version]}"
 end
 
+namespace :pl do
+  desc 'build the gem and place it at the directory root'
+  task :gem_build do
```

Comment:
Maybe this was intentional(?) but the packaging repo already defines the `pl` namespace. Should we use a different namespace so as to not accidentally conflict with `pl::gem`?

Reply:
I was just trying to not interfere with what might possibly be in `bundler/gem_tasks`; that defined a top level `:build` task, so I just defined it in PL to keep away from that. I guess we could just use an arbitrary new namespace to keep away from what the packaging gem defines too, but not sure what that should be called.